lib/src/simulations/lissajous_curve.dart

```diff
@@ -106,7 +106,12 @@ class _LissajousCurveState extends State<LissajousCurve> {
           ),
         ),
       ),
-      bottomNavigationBar: Container(
+      bottomNavigationBar: 
+      Scrollbar(child: SingleChildScrollView(

```

Comment:
You don't need to add single child scroll view here, just wrap listview with scrollbar.